src/common/tvgStr.cpp

```diff
@@ -197,6 +197,8 @@ float strToFloat(const char *nPtr, char **endPtr)
 
 success:
     if (endPtr) *endPtr = (char *)(a);
+    if (val < FLT_EPSILON || val > FLT_MAX) return 0.0f;
```

Comment:
```val < FLT_EPSILON``` is a problem for us?
also ```val > FLT_MAX``` won't catch ```NaN```, right? 
so maybe ```std::isfinite``` ?

Reply:
Thanks! I updated pr